spec/models/reservation_spec.rb

```diff
@@ -413,6 +413,23 @@
             context "there is no following reservation" do
               it_behaves_like "a customer is allowed to edit"
             end
+
+            context "the instrument has a reservation lock window" do
+              before :each do
+                @current_time = Time.zone.now
```

Comment:
This is kind of picky, but I think we should put this in a `let` block instead of an instance variable.